src/style/global.sass

```diff
@@ -31,3 +31,19 @@ ul, ol
     padding-inline: 2rem
     width: 100%
     box-sizing: border-box
+
+.centered-container
+    display: flex
+    width: 100%
+    justify-content: center
+    align-items: center
+
```

Comment:
- Not in use.
- It's already `display: block` so `width: 100%` is not irrelevant.
- Can be simplified with grid:
  ```sass
  display: grid
  justify-content: center
  ```

```suggestion
```